config/betterquesting/DefaultQuests/Quests/PowerfulNuclearP-AAAAAAAAAAAAAAAAAAAAGg==/FasterLutetiumEx-AAAAAAAAAAAAAAAAAAAMbA==.json

```diff
@@ -37,49 +29,13 @@
+      "desc:8": "This quest is late LuV.\n\nYou might have read before that Lutetium is necessary as an input for fusion crafting. In a tier 2 fusion reactor, Lutetium + Chrome \u003d Americium, a required material for many required crafts. Lutetium is a material that is obtained exclusively from breeding, so you\u0027re in the right place to find ways to get it.\n\nThe very first method, EV\u0027s IC2 Thorium nukes, is the cheapest, but also the slowest. Each fuel rod takes 50000 seconds to deplete, almost 14 hours, with half of a Lutetium Dust per depleted rod. Highly stackable designs, such as the 2-chamber 11 quad rod one, will grant you 22 dust per cycle. After building 10 of those, the rate is still 16 dust per hour. This processing speed isn\u0027t good enough to keep progressing forever, and there is no need to make more towers of breeder nukes.\n\nYour 3 main options are:\n\n1) HTGR - fastest but also quite expensive multiblock reactor, unfortunately it requires purified thorium232 you you will need to purify it first\n2) Lutetium bees - if you started bees getting it should be quite easy\n3) Spamming IC2 nuclear reactors"
```

Comment:
the part where it talks about "thorium232" has you you

Reply:
done